src/backend/gl/src/device.rs

```diff
@@ -90,8 +91,8 @@ impl d::Device<B> for Device {
         &self.share.limits
     }
 
-    fn create_heap(&mut self, _: &c::HeapType, _: d::ResourceHeapType, _: u64) -> Result<n::Heap, d::ResourceHeapError> {
-        unimplemented!()
+    fn create_heap(&mut self, heap_type: &c::HeapType, resource_type: d::ResourceHeapType, size: u64) -> Result<n::Heap, d::ResourceHeapError> {
```

Comment:
how do you bear with all those "unused variable warnings"? :)

Reply:
you mean those 'friendly reminders' :>?